Implemented calculator program for assignment 1

kunalaudichya02 opened the pull request.

kunalaudichya02 committed:
Implemented calculator program for assignment 1

ag-devesh commented on calculator.c:
```diff
@@ -0,0 +1,111 @@
+}
+
+//to check the validity of the expression
+int verifyData(const char *expression) {
```
Rename this function

ag-devesh commented on calculator.c:
Empty input giving result as 0. It must be a proper error msg

kunalaudichya02 committed:
Fixed some issues as per review

kunalaudichya02 committed:
Fixed some issues as per review

kunalaudichya02 committed:
Removed calculator.exe from the repository

kunalaudichya02 commented:
Handled empty input: Added a check to ensure the expression is not empty.
Renamed function: Updated the function names
First operator check: Also added validation to check if the expression starts with * or /, which are considered invalid at the beginning.

kunalaudichya02 committed:
some changes

kunalaudichya02 committed:
few changes in the function
